connectors/class-connector-settings.php

```diff
@@ -622,7 +622,15 @@ function( $rule ) use ( $submenu, $record ) {
 	 * @param mixed  $value      Option new value.
 	 */
 	public function callback_update_option( $option, $old_value, $value ) {
-		if ( ( defined( '\WP_CLI' ) && \WP_CLI || did_action( 'customize_save' ) ) && array_key_exists( $option, $this->labels ) ) {
+		if (
+			(
+				( defined( '\WP_CLI' ) && \WP_CLI )
+				||
+				did_action( 'customize_save' )
+			)
+			&&
+			array_key_exists( $option, $this->labels )
+		) {
```

Comment:
<!-- general note -->
**📝 note:** @tharsheblows Can you compare this with the previous version and double-check if I haven't done any error in the logic?

Reply:
@delawski Yes! It looks correct to me.